Frameworks/Foundation/NSDictionary.mm

```diff
@@ -99,8 +99,11 @@ - (NSArray*)allObjects {
         return nil;
     }
 
-    StrongId<NSArray> keyArray = [_keyEnumerator allObjects];
-    StrongId<NSMutableArray> toRet = [NSMutableArray new];
+    StrongId<NSArray> keyArray;
+    keyArray.attach([_keyEnumerator allObjects]);
+
+    StrongId<NSMutableArray> toRet;
```

Comment:
NSMutableArray* toRet =[NSMutableArray array];
..
return toRet; should make things more clear?

Having StrongId is also fine.

---
In reply to: [73973850](https://github.com/Microsoft/WinObjC/pull/851#discussion_r73973850) [](ancestors = 73973850,73973752,73973293)